docs/applications/pyserver/index.rst

```diff
@@ -0,0 +1,16 @@
+Proxy-Server (python)
+=======================================
+
+- a Python server that runs on the Receivers machine
+
+
+
+.. toctree::
+   :maxdepth: 2 
+
+   installation
+   usage
+   cmdline
+   
```

Comment:
Items from`Specifications` should be moved here, after command line options - those specs are proxy-server related.
```
   installation
   usage
   cmdline
   rest-api
   http-headers
   offchain sequence
   components-overview
```